File: app/src/main/java/br/com/alura/ceep/ui/coffemachine/presentation/view/custom/CoffeeViewHolder.java
package br.com.alura.ceep.ui.coffemachine.presentation.view.custom;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;
import androidx.recyclerview.widget.RecyclerView;
import br.com.alura.ceep.ui.coffemachine.R;
import br.com.alura.ceep.ui.coffemachine.domain.CoffeMachineData;
import br.com.alura.ceep.ui.coffemachine.presentation.view.DetailActivity;
import org.jetbrains.annotations.NotNull;

public class CoffeeViewHolder extends RecyclerView.ViewHolder {

  ImageView coffeImage;
  TextView textViewDescription;
  TextView textViewType;
  TextView textViewSize;
  TextView textViewIntensity;
  TextView textQtd;
  TextView textInt;

  public CoffeeViewHolder(@NotNull View itemView) {
    super(itemView);
    coffeImage = itemView.findViewById(R.id.coffeImagetype);
    textViewDescription = itemView.findViewById(R.id.coffeDescription);
    textViewType = itemView.findViewById(R.id.coffeType);
    textViewSize = itemView.findViewById(R.id.coffeSize);
    textViewIntensity = itemView.findViewById(R.id.numberIntensity);
    textQtd = itemView.findViewById(R.id.qtdCoffe);
    textInt = itemView.findViewById(R.id.coffeIntensityTitule);
  }
}

File: app/src/main/java/br/com/alura/ceep/ui/coffemachine/presentation/custom/CoffeeViewHolder.kt
package br.com.alura.ceep.ui.coffemachine.presentation.custom

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import br.com.alura.ceep.ui.coffemachine.R
import br.com.alura.ceep.ui.coffemachine.domain.Coffee
import br.com.alura.ceep.ui.coffemachine.helpers.PhotoHelper
import com.squareup.picasso.Picasso

class CoffeeViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

  var image: ImageView
  var description: TextView
  var name: TextView
  var size: TextView
  var intensity: TextView
  var capsules: TextView

  init {
    image = itemView.findViewById(R.id.image_coffe)
    description = itemView.findViewById(R.id.description)
    name = itemView.findViewById(R.id.name)
    size = itemView.findViewById(R.id.size)
    intensity = itemView.findViewById(R.id.intensity)
    capsules = itemView.findViewById(R.id.capsules)
  }

  fun bind(coffee: Coffee) {
    description.text = coffee.description
    name.text = coffee.name
    size.text = coffee.quantity
    intensity.text = coffee.intensity
    capsules.text = coffee.capsules.toString()
    Picasso.get().load(coffee.image)
      .placeholder(R.drawable.ic_launcher_background)
      .into(image)
  }
}